sdk/core/Azure.Core/src/Response.cs

```diff
@@ -41,6 +41,39 @@ public abstract class Response : IDisposable
         /// </summary>
         public virtual ResponseHeaders Headers => new ResponseHeaders(this);
 
+        /// <summary>
+        /// Gets the contents of HTTP response, if it is available.
+        /// </summary>
+        /// <remarks>
+        /// Throws <see cref="InvalidOperationException"/> when <see cref="ContentStream"/> is not a <see cref="MemoryStream"/>.
+        /// </remarks>
+        public virtual BinaryData Content
+        {
+            get
+            {
+                if (ContentStream == null)
+                {
+                    return new BinaryData(Array.Empty<byte>());
```

Comment:
WE might want to cache this binarydata instance. Maybe add BinaryData.Empty.

Reply:
I will add a `BinaryData.Empty` property and use it.